stats/lib/platform-stats-fetchers.js

```diff
@@ -43,6 +44,28 @@ export const fetchDailyStationAcceptedMeasurementCount = async (pgPool, filter)
   return rows
 }
 
+/**
+ * @param {Queryable} pgPool
+ * @param {import('./typings.js').DateRangeFilter} filter
+ */
+export const fetchTopMeasurementStations = async (pgPool, filter) => {
+  // Block expensive multi-day queries
+  assert(filter.to === filter.from, 400, 'Multi-day queries are not supported for this endpoint')
+
+  const { rows } = await pgPool.query(`
+  SELECT
+    participant_address,
+    COUNT(DISTINCT inet_group) AS inet_group_count,
+    COUNT(DISTINCT station_id) AS station_count,
+    SUM(accepted_measurement_count) AS accepted_measurement_count
+  FROM daily_stations
+  WHERE day >= $1 AND day <= $2
+  GROUP BY participant_address
+  ORDER BY accepted_measurement_count DESC;
```

Comment:
Here is the output of this query for the partial data for today, after landing https://github.com/filecoin-station/spark-evaluate/pull/274.

```
            participant_address             | inet_group_count | station_count | accepted_measurement_count 
--------------------------------------------+------------------+---------------+----------------------------
 pre_column_addition                        |                1 |        154159 |                    1036555
 0xb63153ae08c1d7b4f0dee0b0df725f3a9b8cdaae |              274 |           292 |                       6414
 0x170bCcAe27f8160dDE0CbeF147BDc408d9430e69 |              398 |         33062 |                       3995
 0xc04C7538632a03f7F8Ab651C2363B0641f934612 |              398 |         33066 |                       3877
 0xBA2ff3dA97dF5CA0f8Aa3Fa49F5AbcbD4A5eD36C |              398 |         33036 |                       3832
 0x0aeDc88f2f3aebAE54b9b191E323eEEC0B76c1AF |              114 |           123 |                       2464
 0x7581D288f51622901eaB1d340be773241c53D025 |              129 |           131 |                       2455
 0xCE53e6dEfbD1f0810BFcd691660ad311d0DB24f7 |              107 |           120 |                       2153
 0xB7bAa0811340d2Fc12c7C08eC016993B92C368a9 |               92 |            92 |                       1558
(...)
```

Notice that we treat `inet_group` value `pre_column_addition` as a valid subnet.

And the top participant is the famous `pre_column_addition` 😆 

This won't be a problem from tomorrow & onwards as long as we never look back at the data for today (e.g. if we decide to build data for the last 30 days).

I propose to delete all rows with `pre_column_addition`. We will lose some historical data, but that's not a big deal IMO, as we don't have the full history anyways. In hindsight, we should have deleted those columns as part of the DB schema migration 🙈 

WDYT, @PatrickNercessian @juliangruber?

Reply:
I am always apprehensive toward deleting data, (not that it will break anything, but moreso "maybe we'll want it later!") but that's not a big deal.

An alternative could be something like:

```
  SELECT
    participant_address,
    COUNT(DISTINCT CASE WHEN inet_group != 'pre_column_addition' THEN inet_group END) AS inet_group_count,
    COUNT(DISTINCT station_id) AS station_count,
    SUM(accepted_measurement_count) AS accepted_measurement_count
  FROM daily_stations
  WHERE day >= $1 AND day <= $2
    AND participant_address != 'pre_column_addition'
    AND inet_group != 'pre_column_addition'
  GROUP BY participant_address
  ORDER BY accepted_measurement_count DESC;
```

But again, no hard preference from me.

Reply:
I decided to delete the data with `pre_column_addition`. The SQL query is already too expensive and difficult to optimise via index; let's not make it even more complex & slower.